wikimedia/wikipedia-ios
# For Explore feed options, use form sheet on iPad

Fixes T251564.

**To test**
1. Run app on iPad.
1. Tap the three dots next to an explore feed card.
1. Tap Customize Explore Feed
1. Observe: The new screen (with many switches for various explore feed options) should not cover the entire screen on iPad - it should look like a window within iPad.
1. Repeat steps 2-4 on iPhone. It should work as always - cover nearly the entire screen.

**iPad Screenshots**
![Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-05-13 at 15 20 33](https://user-images.githubusercontent.com/9295855/81872148-0f8aa480-952e-11ea-9b5a-3bffe8680a9e.png) ![Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-05-13 at 15 20 46](https://user-images.githubusercontent.com/9295855/81872155-131e2b80-952e-11ea-8c36-ec65551b4047.png)

**iPhone 11 Max Pro Screenshots**
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 15 17 32](https://user-images.githubusercontent.com/9295855/81872123-039ee280-952e-11ea-929b-df677a939cc8.png) ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 15 17 46](https://user-images.githubusercontent.com/9295855/81872127-0568a600-952e-11ea-9897-131b7fe1bf4d.png)
